### ISSUE TITLE
The plugin seems to use a cache or something

### ISSUE THREAD

**Issue by BenNottelling** [ollydev/jsmpeg-vnc-desktop-duplication#1]
Hey Olly! Remember with your ordinal PR how I said that I was having issues with a black screen on my server? Well, I assumed it was because of something on my server, however, I have a new one now and I'm experiencing the same issue and I think I found the issue with it. 
So, it was working on my server the first time I tried it, but then I rebooted for updates and it wasn't working, I was just getting a black screen, I tried rebooting the server, the app, and doing right click run as admin to avail, but then once when I tried it again I saw this which is from BEFORE I rebooted the server twice!
![issue](https://user-images.githubusercontent.com/17304943/55674124-455a5580-5865-11e9-8c5c-43c022db10d7.PNG)
So I'm assuming that there is a cache somewhere and that's where I'm getting an issue
edit: it went away after a while and is back to normal.
edit 2: came back after another reboot with the same frame.

**Comment by ollydev** [comment]
I never made a new release for a issue I was having when running on a local RDP session. I just uploaded it though, maybe if we're lucky it'll fix your issues?
https://github.com/ollydev/jsmpeg-vnc-desktop-duplication/releases/tag/1.01

I stopped using this since I switched to Linux but i'm interested in fixing this. If the above release doesn't work we'll add some debugging to the desktop duplication to see what's happening.
